Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -21,7 +21,7 @@ static EnableUITKEditor()
             // At the moment, the UITK Asset Editor doesn't have feature parity with the IMGUI version.
             // This is set to false to show the IMGUI version of the InputActionAsset Editor instead.
             // UITK Editor is always be used for the Project Settings Editor regardless of this setting.
-            InputSystem.settings.SetInternalFeatureFlag(InputFeatureNames.kUseUIToolkitEditorForAllAssets, false);
+            InputSystem.settings.SetInternalFeatureFlag(InputFeatureNames.kUseUIToolkitEditorForAllAssets, true);
```

Comment:
Should we remove the feature flag now (and it it's associated code) to tidy that up?

Keeping the feature flag will make it easier to rollback if we are not 100% about it right now. But at some point this should be removed.

Reply:
Can a new flag be added to instead switch to the IMGUI version? Let's say if there's an UITK issue in the future that's blocking some person it would be nice to offer that as a temporary workaround. We could remove it once UITK version has been battle tested properly with the users

Reply:
Good idea - flag renamed in latest comment, logic reversed to be opt in for emergency short term usage and SetInternalFeatureFlag no longer used in package

Reply:
I've also added a changelog